aea/configurations/base.py

```diff
@@ -1397,6 +1402,8 @@ def json(self) -> Dict:
             config["build_entrypoint"] = self.build_entrypoint
 
         # framework optional configs are only printed if defined.
+        if self.required_ledgers is not None:
+            config["required_ledgers"] = self.required_ledgers
```

Comment:
But this is not framework optional. It is required. I guess we change for v1?

Reply:
yes